libdec/arch/riscv.js

```diff
@@ -182,8 +182,8 @@
         instructions: {
             auipc: function(instr) {
                 var dst = instr.parsed.opd[0];
-                var imm32 = parseInt(instr.parsed.opd[1]);
-                var src = instr.location.add(imm32);
+                var imm20 = instr.parsed.opd[1] << 12;
```

Comment:
```suggestion
                var imm20 = parseInt(instr.parsed.opd[1]) << 12;
```
Always convert strings in numbers